corpus/convert.py

```diff
@@ -326,3 +330,69 @@ def run(self):
                     segment.fullname() in segments_whitelist
                 ):
                     f.write(segment.orth + "\n")
+
+
+class CorpusToRawWavHDFJob(Job):
```

Comment:
Discussion question:
is corpus/convert.py the right place? The Job does not convert the corpus (transcript) as the other jobs in this file, but it does convert the collection of audio files (i.e. the corpus?)

Reply:
what about `returnn.hdf`? As this uses the returnn backend to create an HDF file it might fit better there.

Reply:
> what about `returnn.hdf`? As this uses the returnn backend to create an HDF file it might fit better there.

I second this, would also be analog to similar jobs for the `OggZipDataset`.